src/index.js

```diff
@@ -1,6 +1,9 @@
 import reduce from "lodash/reduce";
 import merge from "lodash/merge";
 import has from "lodash/has";
+import get from "lodash/get";
```

Comment:
Now that you're importing so much from `lodash` Any thoughts on just importing all of `lodash` or doing
```javascript
`import { has, get, merge, etc...} from 'lodash';

```